usagov/usagov-2021
# Usagov 1313 sayt federal directories

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1313

## Description

Create a drop down menu for federal agencies.

![Screenshot 2024-02-15 at 3 33 45 PM](https://github.com/usagov/usagov-2021/assets/108532577/ccc3902f-44ca-44dc-a927-090a9d521cdb)


## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Requires New Config

### Requires New Content

### Validation Steps
- You'll need to import config changes so "bin/drush cim"
- On http://localhost/agency-index you should see the search box that's been there
- Now however if you type you should start to see a list of agency suggestions appear. The search space for this list are the agency names, synonyms and acronyms we have for them in Drupal. 
- Clicking on the suggestion should take you to the requested agency page. 
- Clicking outside the suggestion box should hide the box. 

DK provided a list of the most search terms in the box. Please test out at least some of these during the review. 
![Screenshot 2024-02-15 at 3 32 21 PM](https://github.com/usagov/usagov-2021/assets/108532577/d176e325-4360-4de3-9098-b84586b32070)
![Screenshot 2024-02-15 at 3 32 47 PM](https://github.com/usagov/usagov-2021/assets/108532577/4dc40db3-375b-493a-a2d7-c127e1eba0a7)

## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
